### PR TITLE
Feature/cc 0074 get tile color from backend

### PR DESCRIPTION
**Pull request by Kornzk**
get tile color from backend and change mock data

### CODE REVIEW

File: src/components/TopBar/ColorBoard.tsx [hunk 1]
```diff
@@ -1,17 +1,20 @@
-import React from "react";
+import React, { useEffect } from "react";
```
Comment (stevenliu986) [inline src/components/TopBar/ColorBoard.tsx:1]: I didn't find useEffect in your code. Is it needed?

**Review by stevenliu986 — COMMENTED** [review]
Clean unused code.

File: src/components/TopBar/ColorBoard.tsx (hunk 1 above)
Comment (Kornzk) [reply src/components/TopBar/ColorBoard.tsx]: thanks i have deleted it

**Review by stevenliu986 — APPROVED** [review]
LGTM

**Review by Beryl-Zhou — APPROVED** [review]

**Pull request merged**